test/documentation/version.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/* Jongkyu Kim(j.kim@fu-berlin.de), 2016.01.12 */
```

Comment:
Maybe add "Adapted/Adjusted by Encrio Seiler" if you did adapt the script. Or "minor adaptions".